ranger/config/commands.py

```diff
@@ -1859,3 +1859,18 @@ def tab(self, tabnum):
             in sorted(self.modes.keys())
             if mode
         )
+
+
+class display_macro(Command):
+    """:display_macro <marco>
+
+    Show macro contents in the pager.
+    """
+
+    resolve_macros = False
+
+    def execute(self):
+        if not self.arg(1) or self.arg(2):
+            self.fm.notify('Syntax: display_macro <marco>', bad=True)
```

Comment:
Polo!

Reply:
: )

Reply:
Just curious, what does "Polo" mean? I googled it but didn't find an answer :)
That said, when I relooked at the code, I found `Command` class had `args` attribute, from which I could get argument length directly, so I modified the above code to improve the check in the if statement a bit.

Reply:
It's a pool game, someone's blindfolded and yells "Marco!" and then everyone else has to yell "Polo!" and the first one needs to try to find them. Just a joke because you had typo `<marco>` ; )

Reply:
Changed. And thanks for the introduction to the game. It's interesting to know about it.